wandb/__init__.py

```diff
@@ -616,6 +618,17 @@ def init(job_type=None, dir=None, config=None, project=None, entity=None, group=
     if dir:
         os.environ[env.DIR] = dir
         util.mkdir_exists_ok(wandb_dir())
+    resume_path = os.path.join(wandb_dir(), wandb_run.RESUME_FNAME)
+    if resume == True:
+        os.environ[env.RESUME] = "auto"
```

Comment:
I thought it was "always" "allow" or "never"

Reply:
Yeah, I just added "auto" 💯 that will make it write a local file and load from it on the next run.

Reply:
Oh I see.